core/src/test/java/io/aiven/kafka/tieredstorage/chunkmanager/cache/ChunkCacheTest.java

```diff
@@ -243,7 +243,7 @@ void prefetchingWholeSegment() throws Exception {
             chunkCache.configure(Map.of(
                 "retention.ms", "-1",
                 "size", "-1",
-                "prefetching.size", SEGMENT_MANIFEST.chunkIndex().chunks().size()
+                "prefetching.bytes", 20
```

Comment:
It would be nice to bind this `20` to some named constant

Reply:
Fixed